cli/kubernetes/cluster.go

```diff
@@ -12,10 +12,10 @@ import (
 	"github.com/kyokomi/emoji"
 	"github.com/pkg/errors"
 
-	generic "github.com/mudler/kubecfctl/pkg/kubernetes/platform/generic"
-	ibm "github.com/mudler/kubecfctl/pkg/kubernetes/platform/ibm"
-	k3s "github.com/mudler/kubecfctl/pkg/kubernetes/platform/k3s"
-	kind "github.com/mudler/kubecfctl/pkg/kubernetes/platform/kind"
+	generic "github.com/suse/carrier/cli/kubernetes/platform/generic"
+	ibm "github.com/suse/carrier/cli/kubernetes/platform/ibm"
+	k3s "github.com/suse/carrier/cli/kubernetes/platform/k3s"
```

Comment:
👍 went completely unnoticed!

Reply:
Until I tried to fix the issue with the internal/external IPs for k3s and was wondering why my edits to the local files were ignored in the binary. To the point that I added panic statements to see if my code was actually run. It wasn't, and then the :bulb: came on after looking really closely.